protocol/arp.go

```diff
@@ -21,6 +21,9 @@ type ARP struct {
 	IPSrc       net.IP
 	HWDst       net.HardwareAddr
 	IPDst       net.IP
+	// The actual test shows that ARP reply packets sent by the H3C switch include an additional 14 bytes.
+	// This is done by the switch to meet the minimum frame length requirement of 64 bytes.
+	Padding []byte
```

Comment:
No need to define a dedicated field for padding, you can directly update the function `Len()` by adding the corresponding length to meet the minimum frame length requirement of 64 bytes..